pmlc/rt/vulkan/tests/multi-kernel.mlir

```diff
@@ -0,0 +1,73 @@
+// RUN: pmlc-vulkan-runner %s | FileCheck %s
+
+// CHECK: [23,   23,   23],  
+// CHECK: [23,   23,   23],  
+// CHECK: [23,   23,   23]
+
+
+module attributes {gpu.container_module, spv.target_env = #spv.target_env<#spv.vce<v1.0, [Shader], [SPV_KHR_storage_buffer_storage_class]>, {max_compute_workgroup_invocations = 128 : i32, max_compute_workgroup_size = dense<[128, 128, 64]> : vector<3xi32>}>} {
```

Comment:
Can you limit the columns to 80, or 120 at least?

Reply:
Yes, sure.